projects/demo-playwright/tests/legacy/input-phone-international/input-phone-international.pw.spec.ts

```diff
@@ -11,7 +11,7 @@ test.describe('InputPhoneInternational', () => {
 
         test.beforeEach(({page}) => {
             example = new TuiDocumentationPagePO(page).apiPageExample;
-            select = example.locator('tui-input-phone-international select');
+            select = example.locator('tui-textfield .t-ipi-select');
```

Comment:
Let's create `input-phone-international.po.ts` inside [projects/demo-playwright/utils/page-objects](https://github.com/taiga-family/taiga-ui/tree/main/projects/demo-playwright/utils/page-objects):

```ts
import type {Locator} from '@playwright/test';

import {TuiTextfieldWithDataListPO} from './textfield-with-data-list.po';

export class TuiInputPhoneInternationalPO extends TuiTextfieldWithDataListPO {
    public select: Locator = this.host.locator('tui-textfield .t-ipi-select');
}
```

And use it:

```ts
let inputPhoneInternational!: TuiInputPhoneInternationalPO;

test.beforeEach(({page}) => {
    example = new TuiDocumentationPagePO(page).apiPageExample;
    inputPhoneInternational = new TuiInputPhoneInternationalPO(
        example.locator('tui-textfield'),
    );
});

// [...]

await expect(inputPhoneInternational.dropdown).not.toBeAttached();
await inputPhoneInternational.select.click();
```